core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/WorkflowPipelinedRegionsBuilder.scala

```diff
@@ -247,69 +268,94 @@ class WorkflowPipelinedRegionsBuilder(
           }
         }
       }
+
+      // For operators that have only blocking input links
```

Comment:
Maybe it is better to leave a comment here what operators these might be currently or in the future.

Reply:
added examples